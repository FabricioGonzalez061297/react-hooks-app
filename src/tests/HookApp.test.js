import { shallow } from "enzyme";
import {HookApp} from './../HookApp';
import React from 'react'

describe('Prueba en <HookApp />',()=>{
    test('Debe mostrarse correctament',()=>{
        const wrapper = shallow(<HookApp/>);
        expect(wrapper).toMatchSnapshot();
    });
});