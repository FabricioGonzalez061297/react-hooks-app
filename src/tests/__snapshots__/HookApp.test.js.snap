// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en <HookApp /> Debe mostrarse correctament 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
